### PR TITLE
Reviewer Initial In-Browser Visual Design

### PR DESCRIPTION
**Pull request by ricardobaeta**
@bzz @marnovo @dpordomingo @smacker @carlosms 

Here are the goodies ! :)

Some remarks:

1) The tab effect, right now, does not fully translate that the effect of tab navigation will affect "most similar features" & "most disimilar features". It seems like they are disconnected.
2) "Users annotations" & "Similarity score" block should be right aligned to canvas with no margin or padding.
3) The "similarity score" box, should be shorter in width.  
4) Dashboard is still in lowercase
5) Action button to next annotation says "previous". This button only allow to move one forward, and has a secondary button (style equal to "FINISH" button, besides it at the left - per wireframes.
6) Per commented on the code - less file - we need to have (checked) when the radio button is selected, for the styling be properly done. I already added the correct rules for it to look good, they are commented for now.

note: Don't know why but my extra variables added on variables.less didn't appear in the files to stage, nonetheless, here it is @gray-lightish:			 lighten(@gray-base, 56.7%); // #919191

### CODE REVIEW

**Comment by bzz** [comment]
@ricardobaeta nice! 

Just FYI: as with most of GUI changes, attaching a UI screenshot usually speeds up the review.

**Comment by ricardobaeta** [comment]
@bzz Sorry!

image

**Comment by smacker** [comment]
those are my personal preference. But design is a lot about preferences. So I do think I would feel uncomfortable to review 1k file pairs with 10k features in each with those colors. Actually, my eyes might hurt after few minutes, they are very contrasting, I'm not used to it.

File: vendor/github.com/src-d/code-annotation/src/pages/Experiment.less [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@import '../../node_modules/bootstrap/less/variables.less';
```
Comment (smacker) [inline vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: I'm not sure why we see this file here. It shouldn't be here.

File: src/pages/Review.less [hunk 2]
```diff
@@ -62,11 +62,10 @@
...
         flex: 0 0 auto;
         display: block;
-        border-top: 1px solid @gray;
-        border-bottom: 1px solid @gray;
-        margin-bottom: 10px;
-        color: @gray;
+        color: @brand-info;
         cursor: row-resize;
+        background: @gray-darker;
+        padding: 2px 0;
```
Comment (smacker) [inline src/pages/Review.less:132]: I'm pretty sure you didn't push all changes, please check it. This lines can't produce the result we see on the screenshot.

**Comment by marnovo** [comment]
Comments based on the screenshot, not running the PR code:

- White background on the code display UI vs dark overall UI are likely a bad idea over longer periods of time.
- Contrast between lighter and darker rows appear too high, while between the green/red text and the lighter rows appear to low. Adjusting the lighter rows to be darker would solve both problems.

**Comment by ricardobaeta** [comment]
Thanks @marnovo :)

> White background on the code display UI vs dark overall UI are likely a bad idea over longer periods of time.

Totally agree. https://github.com/src-d/code-annotation/issues/68 @smacker showed me an integration with Monokai, looking pretty sweet.

> Contrast between lighter and darker rows appear too high, while between the green/red text and the lighter rows appear to low. Adjusting the lighter rows to be darker would solve both problems.

The force is with you :)

**Comment by ricardobaeta** [comment]
> those are my personal preference. But design is a lot about preferences. So I do think I would feel uncomfortable to review 1k file pairs with 10k features in each with those colors. Actually, my eyes might hurt after few minutes, they are very contrasting, I'm not used to it.

@smacker Thank you, this must be indeed a usable web application for **all** users :) please be so kind to send me a screenshot of your precious coding UI environment.

**Comment by smacker** [comment]
@ricardobaeta I use the standard dark theme from vscode. It's also dark and colorful, but colors are less bright:
screen shot 2018-02-19 at 10 58 29

you can see there are also green & red colors but they are more gentle.

File: src/pages/Review.less (hunk 2 above)
Comment (ricardobaeta) [reply src/pages/Review.less:132]: I've pushed all the changes @smacker

File: vendor/github.com/src-d/code-annotation/src/pages/Experiment.less (hunk 1 above)
Comment (ricardobaeta) [reply vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: Don't know. I started working when I was supposed to, and with the latest version.

**Comment by ricardobaeta** [comment]
@bzz Please be so kind to let me know if I'm blocking this merge in any way.

File: vendor/github.com/src-d/code-annotation/src/pages/Experiment.less (hunk 1 above)
Comment (smacker) [reply vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: you need to fix it.

**Review by smacker — CHANGES_REQUESTED** [review]
there shouldn't be any changes in `vendor` directory

File: vendor/github.com/src-d/code-annotation/src/pages/Experiment.less (hunk 1 above)
Comment (ricardobaeta) [reply vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: I'll do it, but I thought that we had learned from the Annotation PR - where we had conflicts and I had to solve them. We're still figuring out the best way to team up and be productive, but I think that the "time-stamped" version for me to work on, should convey the fact that I need time to do the Visual Design and not making me think of app file structure and changes in the process. What would you suggest, after what I've mentioned?

Comment (smacker) [reply vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: you changed the wrong file. It should be 
```
src/pages/Experiment.less
```

not
```
vendor/github.com/src-d/code-annotation/src/pages/Experiment.less
```

Comment (smacker) [reply vendor/github.com/src-d/code-annotation/src/pages/Experiment.less:1]: or there is something wrong with your local environment. Actually `vendor/github.com/src-d/code-annotation` shouldn't exist at all.

File: src/pages/Review.less (hunk 2 above)
Comment (smacker) [reply src/pages/Review.less:132]: Nope. It's what I see if I checkout to this branch:
screen shot 2018-02-19 at 20 02 30

Comment (ricardobaeta) [reply src/pages/Review.less:132]: @smacker I beg your pardon. I've just pushed what was supposed to be pushed. Now it's all ok for the merge.

image

**Comment by bzz** [comment]
@ricardobaeta in order to be able to merge this, could you please:
 - rebase this on latest master
 - get rid of changes in `vendor/github.com/src-d/code-annotation/src/pages/Experiment.less` as noted by @smacker 

?

Please feel free to ask, here or on slack, in case you need any help to do so.

**Comment by ricardobaeta** [comment]
@bzz Somehow - I really need to improve my git skills to streamline this workflow - I've messed up completely my repo. I've backup all my changes on Review.less and sent the file to @smacker . I already articulated with him, and all there is to do is having Maxim to create a new PR with my .less changes and it will all be good.

This will be an exception to our workflow, and a lesson for me to learn. We will maintain In-Browser Visual Design with a PR. I guarantee that my git skills won't get in the way on my next PR.

I'm closing this PR.

**Pull request closed without merging**